drivers/spi/spi_nrfx_spim.c

```diff
@@ -59,7 +59,7 @@ static inline nrf_spim_frequency_t get_nrf_spim_frequency(uint32_t frequency)
 		return NRF_SPIM_FREQ_2M;
 	} else if (frequency < 8000000) {
 		return NRF_SPIM_FREQ_4M;
-#if defined(CONFIG_SOC_NRF52833) || defined(CONFIG_SOC_NRF52840)
+#if defined(CONFIG_SOC_NRF52833) || defined(CONFIG_SOC_NRF52840) || defined(CONFIG_SOC_NRF5340_CPUAPP)
```

Comment:
could you break the line with a '\' ? 

lgtm anyway

Reply:
Hello @tbursztyka 
Thanks for the comment, would you mean split the change into two lines like:
```
#if defined(CONFIG_SOC_NRF52833) || defined(CONFIG_SOC_NRF52840) \
                                 || defined(CONFIG_SOC_NRF5340_CPUAPP)
```

Reply:
yes please